pkg/auth/login/login.go

```diff
@@ -53,6 +52,7 @@ func (a *AuthorizationCodeGrant) Execute(ctx context.Context, ssoCfg *SSOConfig,
 	if err := a.loginMAS(ctx, masSSOCfg); err != nil {
 		return err
 	}
+	a.Logger.Info(a.Localizer.MustLocalize("login.log.info.loggedIn"))
```

Comment:
@wtrocki , @craicoverflow 

I've moved this here because we may end up in a situation like this

Logging in...

Logged in successfully
Logging in to identity.api.openshift.com...
Error: login operation took too long. Please try again. Run the command in verbose mode using the -v flag to see more information

After I did that, everything looks much better

Logging in...

Logging in to identity.api.openshift.com...
Error: login operation took too long. Please try again. Run the command in verbose mode using the -v flag to see more information

Reply:
```go
a.Logger.Info(a.Localizer.MustLocalize("login.log.info.loggingInMAS", localize.NewEntry("Host", masSSOHost)))
```

Could you make this into a debug message instead? The user does not need to see two login statuses whyen it should happen automatically. The new ordering might be a bit confusing but that's just an effect of repeated login messages